physics/clm_lake.f90

```diff
@@ -498,14 +501,14 @@ SUBROUTINE clm_lake_run( &
 
         wght2 = day_of_month/month_length(month)
         if(wght2<0 .or. wght2>1) then
-          if(lakedebug) then
+          if(debug_print) then
             write(0,*) 'Warning: wght2 is not 0..1: ',wght2
           endif
           wght2 = max(0.0_kind_lake,min(1.0_kind_lake,wght2))
         endif
         wght1 = 1.0_kind_lake - wght2
   
-        if(LAKEDEBUG .and. me==0) then
+        if(debug_print ) then
```

Comment:
Do we want to do this on all processors?

Reply:
That lets us confirm the value is the same, and correct, on all ranks.